app/libs/classes/viva_my_pages.py

```diff
@@ -42,6 +42,13 @@ def get_workflow_list(self):
         except KeyError:
             raise Fault(message='No workflows found', code=404)
 
+    def get_workflow_latest(self):
+        try:
+            person_caseworkflow = self._get_person_caseworkflow(limit=1)
+            return person_caseworkflow['vivadata']['vivacaseworkflows']['workflow']
+        except KeyError:
+            raise Fault(message='No workflow latset found', code=404)
```

Comment:
This error message has incorrect english grammar.
Suggestion: "There was no workflow to be found" or "No workflow was found"